app/components/ui/Cards/OppEventCard.tsx

```diff
@@ -1,35 +1,41 @@
 import React from "react";
 import { Button } from "components/ui/inline/Button/Button";
+import { StrapiModel } from "models/Strapi";
 import styles from "./OppEventCard.module.scss";
-import { OppEventDate } from "./OppEventDate";
+import { FormattedDate } from "./FormattedDate";
 
 interface OppEventCardProps {
-  slug: string;
-  image: string;
-  title?: string;
-  startDate?: Date;
-  endDate?: Date;
+  details: {
+    id: string;
+    imageUrl: string | null;
+    title: string;
+    calendarEvent: StrapiModel.CalendarEvent;
+  };
```

Comment:
I think about component props similarly to function parameters -- components are just functions after all.  _More than two_ can be a code smell for a missing configuration object or too much responsibility. What's nice about configuration objects like this is that 1) you get a useful namespace and 2) preparation of this object in upstream code can be simplified into another single responsibility function or statement.